src/main/resources/application-hsqldb.yml

```diff
@@ -0,0 +1,12 @@
+app.db.url: jdbc:hsqldb:mem:data-compliance-db;sql.syntax_pgs=true;shutdown=false
+
+spring:
+  main:
+    allow-bean-definition-overriding: true
+  jpa:
+    show-sql: true
+    hibernate.ddl-auto: create-drop
+  flyway:
+    enabled: true
+    locations: classpath:db/migratio
```

Comment:
Is that a typo?

Reply:
Hmmm it didn't stop the tests from passing - maybe you don't need it at all?

Reply:
Yup it is a typo & I have no idea how the tests are still passing. When I exclude 'quartz' which is in the same directory, It is excluded by flyway

Reply:
Ah I see, the default location is db/migration. As I did not provide a valid path, it defaulted to that.

https://flywaydb.org/documentation/configuration/parameters/locations

Reply:
I'd prefer to explicitly provide it, instead of omit it even it results in the same location.

I have updated it